modules/weko-schema-ui/weko_schema_ui/schema.py

```diff
@@ -820,6 +820,33 @@ def merge_json_xml(json, dct):
                     k = 'jpcoar:file'
                 elif k == 'custom:system_identifier':
                     k = 'jpcoar:identifier'
+
+                # Remove items that are not set as controlled vocabulary
+                elif k == 'jpcoar:creator' or k == 'jpcoar:contributor' \
+                        or k == 'jpcoar:rightsHolder':
+                    if '@attributes' in v['jpcoar:nameIdentifier']:
+                        lst_nameIdentifierScheme = v['jpcoar:nameIdentifier'][
+                            '@attributes']['nameIdentifierScheme'][0]
+                        if '@value' in v['jpcoar:nameIdentifier']:
+                            lst_value = v['jpcoar:nameIdentifier']['@value'][0]
+                        lst_nameIdentifierURI = v['jpcoar:nameIdentifier'][
+                            '@attributes']['nameIdentifierURI'][0]
+                        index_remove_items = []
+
+                        for identifior_item in lst_nameIdentifierScheme:
+                            if not identifior_item.upper() in current_app.\
```

Comment:
I think we need using extractly capitalization from: https://schema.irdb.nii.ac.jp/en/schema/3-1.

Thanks,